.github/workflows/fetch-merged-prs.yml

```diff
@@ -0,0 +1,67 @@
+name: Fetch Merged PRs After Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      release:
+        description: 'Release after which to fetch merged PRs'
+        required: true
+      branch:
+        description: 'Branch to fetch merged PRs from'
+        required: true
+        default: 'main'
+
+jobs:
+  fetch-merged-prs:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Get Release Date
+        id: get-release-date
+        run: |
+          RELEASE_DATE=$(curl -H "Accept: application/vnd.github.v3+json" -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" "https://api.github.com/repos/$${{ github.repository }}/releases/tags/${{ github.event.inputs.release }}" | jq -r '.published_at')
+          echo "RELEASE_DATE=$RELEASE_DATE" >> $GITHUB_ENV
+
+      - name: Fetch merged PRs after release
+        id: fetch-prs
+        run: |
+          echo "PR Title,PR Link,PR Owner,Jira Task URL" > merged_prs.csv
+
+          PAGE=1
+
+          while : ; do
+            response=$(curl -H "Accept: application/vnd.github.v3+json" \
+              -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
+              "https://api.github.com/repos/$${{ github.repository }}/pulls?state=closed&base=${{ github.event.inputs.branch }}&per_page=100&page=$PAGE")
```

Comment:
Shouldn't it be 'state=merged'?

Reply:
The Github API doesn't have that parameter unfortunately. "Closed" returns all closed merged and unmerged PRs, and then we filter out unmerged ones.